server/src/main/scala/org/http4s/server/middleware/FormDataDecoder.scala

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s
+package server
+package middleware
```

Comment:
This isn't really a middleware, which are `Http => Http` or `Client => Client`.  Would core be a more appropriate place for this, I wonder?

Reply:
moved to `core`.